python/ray/actor.py

```diff
@@ -23,6 +24,22 @@
 
 DEFAULT_ACTOR_METHOD_NUM_RETURN_VALS = 1
 
+_actor_init_error = None
```

Comment:
This should be a field in the worker class instead of a global variable.

Reply:
Fixed